app/Scrambler.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable-all
+
+from random import choice, shuffle
+
+from Picture import Picture
+from Tile import Tile
+
+
+class Scrambler:
+    @classmethod
+    def scramble_rows(cls, picture: Picture) -> None:
+        """Splits an Image up into rows an rearranges them."""
+        num_tiles = 8
+        image_size = picture.image.size
+        tile_size = (image_size[0], int(image_size[1] / num_tiles))
+
+        for y in range(num_tiles):
+            x0, y0, x1, y1 = (0, tile_size[1] * y, tile_size[0], tile_size[1] * (y + 1))
+            picture.tiles[y] = Tile(picture.image.crop((x0, y0, x1, y1)), (x0, y0))
+
+        picture.tile_order = list(picture.tiles.keys())
+        shuffle(picture.tile_order)
+
+    @classmethod
+    def scramble_grid(cls, picture: Picture) -> None:
+        """Splits an Image up into tiles an rearranges them and rotates each tile randomly."""
+        num_of_tiles = 4
```

Comment:
The number of tiles could be a parameter to make the method more reusable. What do you think? It would require some additional changes since your logic assumes 4 tiles.

The method signature would look something like this:  

```python
def scramble_grid(cls, picture: Picture, num_of_tiles: int = 4 }) -> None:
    ...
```

Reply:
Made the changes as expected